csunplugged/tests/topics/loaders/test_curriculum_integrations_loader.py

```diff
@@ -12,20 +14,45 @@ class CurriculumIntegrationsLoaderTest(BaseTestWithDB):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.test_data = TopicsTestDataGenerator()
-        self.loader_name = "curriculum_integrations"
+        self.base_path = os.path.join(self.test_data.LOADER_ASSET_PATH, "curriculum_integrations")
 
     def test_basic_config(self):
-        config_file = os.path.join(self.loader_name, "basic-config.yaml")
+        config_file = "basic-config.yaml"
+
+        self.test_data.create_curriculum_area("1")
+        topic = self.test_data.create_topic("1")
+
+        ci_loader = CurriculumIntegrationsLoader(topic, base_path=self.base_path, structure_filename=config_file)
+        ci_loader.load()
+
+        ci_objects = CurriculumIntegration.objects.all()
+
+        self.assertEqual(1, len(ci_objects))
+
+        ci = ci_objects[0]
+        self.assertSetEqual(set(["en"]), set(ci.languages))
+        self.assertEqual("English Heading", ci.name)
+        self.assertIn("English curriculum integration content", ci.content)
+
+    def test_translation(self):
+        config_file = "translation.yaml"
 
         self.test_data.create_curriculum_area("1")
         topic = self.test_data.create_topic("1")
 
-        ci_loader = CurriculumIntegrationsLoader(config_file, topic, self.test_data.LOADER_ASSET_PATH)
+        ci_loader = CurriculumIntegrationsLoader(topic, base_path=self.base_path, structure_filename=config_file)
         ci_loader.load()
 
         ci_objects = CurriculumIntegration.objects.all()
 
-        self.assertQuerysetEqual(
-            ci_objects,
-            ["<CurriculumIntegration: Integration 1>"]
-        )
```

Comment:
why are we removing this check that Queryset is as expected?

Reply:
The test has essentially just been broken into more detailed checks in the lines below - I check only one object has been returned, and that the heading and content are as expected.